File: JavaCore/Q2.java
package JavaCore;

public class Q2 {

    public static void main(String[] args) {
        //Q2. Write a program to find the number of occurrences of the duplicate words in a string and print them ?
        String s = "Hello world hello this world is beautiful hello boy";
        String lower = s.toLowerCase();
        String ss[]= lower.split(" ");
        for(int i=0; i<ss.length; i++){
            int counter = 1;
            for(int j=i+1; j<ss.length;j++){
                if (ss[i].equals(ss[j])){
                    counter++;

                    ss[j] = "0";
                }
            }


            if (counter > 1 && ss[i]!="0"){
                System.out.println("Word: " + ss[i] + "  Occurences: "+ counter);
            }
        }



    }

}

File: Kotlin_1/src/exercise1/Q2.kt
package exercise1
//Q2. Write a program to find the number of occurrences of the duplicate words in a string and print them.
fun main(){
    val str = readLine().toString()
    val lowerStr = str.toLowerCase()
    val arr = lowerStr.split(" ").toTypedArray()

    for (i in arr.indices) {
        var duplicate = 1
        for (j in i+1 until arr.size) {
            if (arr[i] == arr[j]) {
                duplicate++
                arr[j] = "0"
            }
        }
        if (duplicate > 1 && arr[i] != "0")
            println("Word: " + arr[i] + " \nOccurences: " + duplicate)
    }
}

